client/src/app/metadata-details-dialog/metadata-details-dialog.component.ts

```diff
@@ -4,14 +4,12 @@ import {MetadataFormLayout, MetadataFormTab} from '../metadata-schema-form/model
 import {ActivatedRoute} from '@angular/router';
 import {IngestService} from '../shared/services/ingest.service';
 import {SchemaService} from '../shared/services/schema.service';
-import {concatMap, map, tap} from 'rxjs/operators';
 import {MAT_DIALOG_DATA, MatDialogRef} from '@angular/material/dialog';
 import {LoaderService} from '../shared/services/loader.service';
 import {MetadataDocument} from '../shared/models/metadata-document';
-import {Observable, of} from 'rxjs';
-import {ListResult} from '../shared/models/hateoas';
 import {MetadataFormComponent} from '../metadata-schema-form/metadata-form/metadata-form.component';
 import {AlertService} from '../shared/services/alert.service';
+import * as _ from 'lodash';
```

Comment:
This might happen in a few places but it's better to import the specific function you need (e.g. `import map from 'lodash/map'`) so you don't send bundle the whole library. This way it will tree-shake but I think if you import with this syntax it wont

Reply:
makes sense! thanks!